Sources/SwerverCore/Router/Router.swift

```diff
@@ -10,27 +10,29 @@ public class Router {
         populateRoutes()
     }
 
-    public func process(request: HttpRequest) -> String {
-
-        let targetURL = request.getUrl()
-        let targetMethod = request.getMethod()
-
-        if routes.routeExist(url: targetURL) {
-
-            let targetRoute = routes.fetchRoute(url: targetURL)
-
-            guard let action = targetRoute[targetMethod] else {
-                return "HTTP/1.1"
+    public func process(request: HttpRequest?) -> HttpResponse {
+
+        if let request = request {
+            let targetURL = request.getUrl()
+            let targetMethod = request.getMethod()
+            
+            if routes.routeExist(url: targetURL) {
+                
+                let targetRoute = routes.fetchRoute(url: targetURL)
+                
+                guard let action = targetRoute[targetMethod] else {
+                    return notFoundAction.execute()
+                }
+                return action.execute()
             }
-            return responseHeaderFormatter.format(httpResponse: action.execute())
         }
-        return responseHeaderFormatter.format(httpResponse: notFoundAction.execute())
+        return notFoundAction.execute()
```

Comment:
Nitpick: I would put this in an `else` branch. It makes it slightly more explicit that these branches are mutually exclusive.

Reply:
will fixed that